src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -203,6 +203,10 @@ def product_version(self):
 _session = TelemetrySession()
 
 
+def has_exceptions():
+    return len(_session.exceptions) > 0
```

Comment:
this could be serivce error or client error, right?

Reply:
yes, could be any error.

Reply:
What type of this `exceptions`? What's the content of the `exceptions`?

Reply:
`has_exceptions` is defined outside `TelemetrySession` ? How can below `telemetry.has_exceptions` work ?

Reply:
`_session` is a shared instance of `TelemetrySession` in `telemetry.py`, so you can do `from azure.cli.core import telemetry` and just call `telemetry.has_exceptions`

Reply:
> What type of this `exceptions`? What's the content of the `exceptions`?

This contains all sorts of exceptions that are added to the result of a command. It could be CLIError, CloudError or others. It usually contains an error message, please see the PR description for an example.